task_15/task_2.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+
+let a = 20;
+let b = 15;
+let temp;
+
+temp = a;
+a = b;
+b = temp;
+
+console.log(a);
+console.log(b);
+
+let a = 20;
```

Comment:
You initialize the same variable in one file!
Error will be thrown

Reply:
correct